### DISCUSSION TITLE
SASjs Lint| add more rules

### DISCUSSION THREAD

**Discussion by Masum-pixel** [sasjs/lint#183] [Ideas]
Hi Team,
Thank you for updating few rules in SASjs Lint. Could you please add more rule for best practices of SAS Code ?
Please check this paper for your reference: 
1. https://www.lexjansen.com/pharmasug/2022/AP/PharmaSUG-2022-AP-072.pdf
2. https://cpb-us-w2.wpmucdn.com/sites.wustl.edu/dist/8/1443/files/2015/10/Optimize-Your-Code.pdf

**Comment by allanbowe** [comment]
Hi @Masum-pixel - thanks for bringing up this discussion point.  We'd love to add more rules, however our team is small (https://sasapps.io/about) and, unfortunately, resource constrained.  

Would you be interested in contributing?  I'd be happy to arrange a walkthrough of the development process for you.

Alternatively, would your organisation be interested in working with us on a commercial project or trial of Data Controller for SAS?  The revenues we make from these are funnelled directly back into the SASjs framework.  

In the meantime we will certainly consider the documents you have provided, when we next get a customer who is interested in supporting this part of the framework, or if a sponsor is found.

**Comment by Masum-pixel** [comment]
Hi,

Currently we are working on it.But we have no experience in typescipt.Sure
we will try to understand the logic.But if you connect through virtual meet
that better so that we can discuss.Also if you suggest something  on the
same.
Thank you.
